src/pysdmx/model/category.py

```diff
@@ -6,12 +6,11 @@
 
 from typing import Iterator, List, Optional, Sequence, Set
 
-from msgspec import Struct
-
+from pysdmx.model import Item, ItemScheme
 from pysdmx.model.organisation import DataflowRef
 
 
-class Category(Struct, omit_defaults=True):
+class Category(Item, frozen=False):  # type: ignore[misc]
```

Comment:
Why has omit_defaults been removed?

Reply:
Solved on new commit. Just forgot when changing the class heirarchy.